crates/commonware-node/src/dkg/manager/actor/mod.rs

```diff
@@ -272,11 +238,7 @@ where
         let mut tx = DkgReadWriteTransaction::new(self.db.read_write());
 
         // Emits an error event on return.
-        if self.post_allegretto_init(&mut tx).await.is_err() {
-            return;
-        }
-        // Emits an error event on return.
-        if self.pre_allegretto_init(&mut tx).await.is_err() {
+        if self.initialize_epoch_state(&mut tx).await.is_err() {
```

Comment:
i forget, why dont we `?`

Reply:
lets log it at least?

Reply:
But we do! See the definition:
```rust
    #[instrument(skip_all, err)]
    pub(super) async fn initialize_epoch_state(
```

~~I am going to add back the comment.~~
The comment is still above the line. Was confused by the diff.